src/inc-lib-init.php

```diff
@@ -1084,336 +1084,488 @@ function lovd_getInstallURL ($bFull = true)
     if ($sTranscriptID && !isset($aTranscriptOffsets[$sTranscriptID])) {
         $aTranscriptOffsets[$sTranscriptID] = $_DB->query('SELECT position_c_cds_end FROM ' . TABLE_TRANSCRIPTS . ' WHERE (id = ? OR id_ncbi = ?)',
             array($sTranscriptID, $sTranscriptID))->fetchColumn();
         if (!$aTranscriptOffsets[$sTranscriptID]) {
-            // Transcript not configured correctly.
-            // Don't die here; we might not even need these positions. We'll die later if we do.
+            // The transcript is not configured correctly. We will treat this transcript as unknown.
             $sTranscriptID = '';
         }
-    } elseif ($sTranscriptID === false) {
+
+    } elseif ($sTranscriptID == false) {
         // If the transcript ID is passed as false, we are asked to ignore not having the transcript.
         // Some random number, high enough to not be smaller than position_start if that's not in the UTR.
-        $aTranscriptOffsets[0] = 1000000;
-    }
-
-    // Isolate the position(s) from the variant. We don't support combined variants.
-    // We're not super picky, and would therefore approve of c.1_2A>C; we also
-    //  don't check for the end of the variant, it may contain bases, or not.
-    if (preg_match('/^([cgmn])\.(\()?([\-\*]?\d+)([-+](?:\d+|\?))?(?:_([\-\*]?\d+)([-+](?:\d+|\?))?)?([ACGT]>[ACGT]|con|del(?:ins)?|dup|inv|ins|\|(?:gom|lom|met=)|=)(.*)(?(2)\))/', $sVariant, $aRegs)) {
-        //             1 = Prefix; indicates what kind of positions we can expect, and what we'll output.
-        //                       2 = Do we have an opening parenthesis?
-        //                            3 = Start position, might be negative or in the 3' UTR.
-        //                                        4 = Start position intronic offset, if available.
-        //                                                             5 = End position, might be negative or in the 3' UTR.
-        //                                                                         6 = End position intronic offset, if available.
-        //                                                                                            7 = The variant, which we'll use to determine the type.
-        //                                                                                                                                 8 = The suffix.
-        list(, $sPrefix,, $sStartPosition, $sStartPositionIntron, $sEndPosition, $sEndPositionIntron, $sVariant, $sSuffix) = $aRegs;
-
-        if ($bCheckHGVS) {
-            // This was quite a lossy check, sufficient to get positions and type, but we need a HGVS check now.
-            if ($sVariant == 'con') {
-                // Conversion has been removed in favor of delins.
-                // See: http://varnomen.hgvs.org/bg-material/consultation/svd-wg009/
-                return false;
+        $aTranscriptOffsets[$sTranscriptID] = 1_000_000;
```

Comment:
Vanwaar deze underscores?

Reply:
Voor de leesbaarheid! Ik vind het persoonlijk fijn om direct te kunnen zien hoeveel nullen erin zitten. Je mag het weghalen als je het zelf niet mooi/fijn vindt, maar ik zou er zelf altijd liever voor kiezen dit wel te gebruiken in mijn eigen code.

Reply:
Helaas kan dit niet voor 7.4, ik krijg dus een parse error. Als 1000000 een betekenis zou hebben, zou je er 1*1000*1000 van kunnen maken, maar in dit geval betekent het helemaal niets behalve "een absurd hoog getal", dus graag terugdraaien naar 1000000 :wink: